src/Compilers/Core/MSBuildTask/Microsoft.CSharp.Core.targets

```diff
@@ -198,5 +198,7 @@
     <CommandLineArgsForDesignTimeEvaluation>-langversion:$(LangVersion)</CommandLineArgsForDesignTimeEvaluation>
     <CommandLineArgsForDesignTimeEvaluation Condition="'$(ChecksumAlgorithm)' != ''">$(CommandLineArgsForDesignTimeEvaluation) -checksumalgorithm:$(ChecksumAlgorithm)</CommandLineArgsForDesignTimeEvaluation>
     <CommandLineArgsForDesignTimeEvaluation Condition="'$(DefineConstants)' != ''">$(CommandLineArgsForDesignTimeEvaluation) -define:$(DefineConstants)</CommandLineArgsForDesignTimeEvaluation>
+    <CommandLineArgsForDesignTimeEvaluation Condition="'$(Features)' != ''">$(CommandLineArgsForDesignTimeEvaluation) -features:$(Features)</CommandLineArgsForDesignTimeEvaluation>
+    <CommandLineArgsForDesignTimeEvaluation Condition="'$(DocumentationFile)' != ''">$(CommandLineArgsForDesignTimeEvaluation) -doc:$(DocumentationFile)</CommandLineArgsForDesignTimeEvaluation>
```

Comment:
This might need some quoting. Now, ironically, if all the IDE cares about is if it's set or not set, then it's kinda fine if it doesn't get quoting and only the first part of the file name is treated as part of the switch. But we should probably either quote, or leave a comment here that we don't actually care.